Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -77,6 +77,11 @@ void PPCSymbolDB::AddKnownSymbol(u32 startAddr, u32 size, const std::string& nam
     {
       PPCAnalyst::AnalyzeFunction(startAddr, tf, size);
       checksumToFunction[tf.hash] = &(functions[startAddr]);
+
+      tf.function_name = tf.name.substr(0, tf.name.find('('));
```

Comment:
Fixed; I moved this to GetStrippedFunctionName.

By the way, is it useless to return by const value, as is done [here](https://github.com/dolphin-emu/dolphin/pull/4250/commits/13337c19ccdfb30f5d6fd6edac54cd0a593f7478#diff-642499a338bb8fdd8fbc6db230363c14R34)?